clean/post_pprr.py

```diff
@@ -11,7 +11,11 @@
 
 def standardize_agency(df):
     df.loc[:, "agency"] = df.agency.str.strip()\
-        .str.lower().fillna("").str.replace(r"(\w)\.\s*(\w)\.", r"\1\2")
+        .str.lower().fillna("").str.replace(r"(\w)\.\s*(\w)\.", r"\1\2", regex=True)\
+        .str.replace(r'e\. baton rouge so', 'baton rouge so', regex=True)\
+        .str.replace(r'e\. jefferson levee pd', 'east jefferson levee pd', regex=True)\
+        .str.replace(r'^st ', 'st. ', regex=True)\
+        .str.replace(' parish', '', regex=False)
```

Comment:
POST data should share the same agency text as the current output data which means:
- Agency should be in title case.
- Agency should end with either SO or PD (is that all the abbreviations?).
- We should remove all the agency assignments in all match scripts that read POST data. After all, the agency text should be correct at the clean stage.
- And finally, we should hunt down any deviation from this formula in the `fuse/all.py` script, make sure it raises an error if it finds an agency that is not found in POST data. Then we should update this `clean/post_pprr.py` script with the correct agency text.

Reply:
@pckhoi 
Pushed new code, please review when you have the time. 

- Abbreviations have been standardized as best as possible, i.e., PD, SO, and DA (for District Attorney's Offices in cases where we receive brady lists) have been standardized. There are some agencies that I'm still not sure how to abbreviate at the moment, like: "Abbeville City Marshall". I think that it's fine to leave them as is because we only intend to collect data from PDs, SOs, and DAs for the foreseeable future. 

- When I remove the agency assignments from match scripts that read POST data I find that officers are matched to officers from different agencies, i.e., "John Smith" from Baton Rouge SO is matched to "John Smith" from Lafayette PD instead of "John Smith" from Baton Rouge SO.  Thoughts? 

- Added a function that raises an error if an agency is in the event_df and not in the post_event_df

Reply:
Very nice! Re mismatching of "John Smith", I think it's probably a typo. I can't think of any valid reason.

Reply:
I completely misunderstood what you meant here: "We should remove all the agency assignments in all match scripts that read POST data. After all, the agency text should be correct at the clean stage." Sorry! Corrections are in process.